Adds windows installation documentation (#1825)
opensearch-project/documentation-website: _dashboards/install/windows.md

---
layout: default
title: Windows
parent: Install OpenSearch Dashboards
nav_order: 37
---

# Run OpenSearch Dashboards on Windows

Follow the steps below to install OpenSearch Dashboards on Windows.

Make sure you have a zip utility installed.
{: .tip }

1. Download the [`opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip`](https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip){:target='\_blank'} archive.

1. To extract the archive contents, right-click to select **Extract All**.

1. Run OpenSearch Dashboards.

   There are two ways of running OpenSearch Dashboards:

>    1. Run the batch script using the Windows UI:

Reviewer: Windows Desktop Environment?
Not really sure how to call it as well, windows does categorize them as desktop vs server.

>    1. Run the batch script using the Windows UI:

Author: I'll spell out user interface

      1. Navigate to the top directory of your OpenSearch Dashboards install and open the `opensearch-dashboards-{{site.opensearch_version}}` folder.
      1. If desired, modify `opensearch_dashboards.yml` located in the `config` folder.
      1. Open the `bin` folder and run the batch script by double-clicking the `opensearch-dashboards.bat` file. This opens a command prompt with an OpenSearch Dashboards instance running.

   1. Run the batch script from a command prompt or Powershell:

>       1. Open command prompt by entering `cmd`, or Powershell by entering `powershell`, in the search box next to **Start** on the taskbar. 

Author: I am referring to the application, so I'll capitalize all instances of Command Prompt.

---
layout: default
title: Windows
parent: Install OpenSearch Dashboards
nav_order: 37
---

# Run OpenSearch Dashboards on Windows

> Follow the steps below to install OpenSearch Dashboards on Windows.

Reviewer 2: ```suggestion
Perform the following steps to install OpenSearch Dashboards on Windows.
```

> Make sure you have a zip utility installed.

Reviewer 2: "ZIP"?

Author: Normally, zip is not capitalized.

{: .tip }

1. Download the [`opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip`](https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip){:target='\_blank'} archive.

1. To extract the archive contents, right-click to select **Extract All**.
   

>    Note: Some versions of the Windows operating system limit the file path length. If you encounter a path length-related error when unzipping the archive, perform the following steps to enable long path support:

Reviewer 3: Can you indent 4 times and use the {: .note } tag to put this in that formatting? Or is the note intended to be expressed like this? I think indenting 4x preserves the numbered list.
"path-length related error"

Author: I will bold the word "note"

   1. Open Powershell by entering `powershell` in the search box next to **Start** on the taskbar. 
   1. Run the following command in Powershell:
      ```bat
      Set-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem LongPathsEnabled -Type DWORD -Value 1 -Force
      ```
   1. Restart your computer.

1. Run OpenSearch Dashboards.

   There are two ways of running OpenSearch Dashboards:

>    1. Run the batch script using the Windows user interface:

Reviewer 2: ```suggestion
   1. Run the batch script using the Windows UI:
```

>       1. Navigate to the top directory of your OpenSearch Dashboards install and open the `opensearch-dashboards-{{site.opensearch_version}}` folder.

Reviewer 2: ```suggestion
      1. Navigate to the top directory of your OpenSearch Dashboards installation and open the `opensearch-dashboards-{{site.opensearch_version}}` folder.
```

>       1. If desired, modify `opensearch_dashboards.yml` located in the `config` folder to change the default OpenSearch Dasbhoards settings.

Reviewer 2: ```suggestion
      1. If desired, modify `opensearch_dashboards.yml`, located in the `config` folder, to change the default OpenSearch Dashboards settings.
```

      1. Open the `bin` folder and run the batch script by double-clicking the `opensearch-dashboards.bat` file. This opens a command prompt with an OpenSearch Dashboards instance running.

   1. Run the batch script from Command Prompt or Powershell:

      1. Open Command Prompt by entering `cmd`, or Powershell by entering `powershell`, in the search box next to **Start** on the taskbar. 

>       1. Change to the top directory of your OpenSearch Dashboards install.

Reviewer 2: ```suggestion
      1. Change to the top directory of your OpenSearch Dashboards installation.
```
